cmd/sharded-test-server/virtual.go

```diff
@@ -88,9 +88,9 @@ func newVirtualWorkspace(ctx context.Context, index int, servingCA *crypto.CA, h
 		return nil, fmt.Errorf("failed to create vw client cert: %w", err)
 	}
 
-	servingCAPath, err := filepath.Abs(filepath.Join(workDirPath, ".kcp/serving-ca.crt"))
+	servingCAPath, err := filepath.Abs(filepath.Join(workDirPath, ".kcp/shard-serving-ca.crt"))
 	if err != nil {
-		return nil, fmt.Errorf("error getting absolute path for %q: %w", filepath.Join(workDirPath, ".kcp/serving-ca.crt"), err)
+		return nil, fmt.Errorf("error getting absolute path for %q: %w", filepath.Join(workDirPath, ".kcp/shard-serving-ca.crt"), err)
```

Comment:
I was unclear about that - indeed the e2e tests are failing because `admin.kubeconfig` can't connect to the VW servers directly when the CA certs are decoupled - do we expect users to connect directly to the VW server endpoints, or only via the proxy?

Reply:
They connect directly. There could potentially be a proxy in between. But the paths used for different services must be non-overlapping.

Reply:
I removed the splitting of CA certs for now, will revisit as a follow-up